app/components/provider_interface/application_card_component.rb

```diff
@@ -4,9 +4,9 @@ class ApplicationCardComponent < ViewComponent::Base
 
     attr_accessor :accredited_provider, :application_choice, :application_choice_path,
                   :candidate_name, :course_name_and_code, :course_provider_name, :changed_at,
-                  :most_recent_note, :site_name_and_code
+                  :most_recent_note, :site_name_and_code, :sort_by
 
-    def initialize(application_choice:)
+    def initialize(application_choice:, sort_by: nil)
```

Comment:
We could rename this to conceptually decouple the "sorting" from how we display the date:
 
```suggestion
    def initialize(application_choice:, show_date:)
```

Where `show_date` can be either `:last_updated` or `:reject_by_default`. This would also removes the magic string in the `contextual_date` method.

Reply:
thanks @tijmenb amended here https://github.com/DFE-Digital/apply-for-teacher-training/pull/2426/commits/6ccfeb16667e05507b46545f2bce0c014f55ee78